src/site/content/en/accessible/color-and-contrast/index.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: Color and contrast
+authors:
+  - dgash
+  - megginkearney
+  - rachelandrew
+  - robdodson
+date: 2020-05-28
+description: |
+  If you have good vision, it's easy to assume that everyone perceives colors, or text legibility, the same way you do — but of course that's not the case.
+tags:
+  - blog
+  - accessibility
+---
+
+If you have good vision, it's easy to assume that everyone perceives colors,
+or text legibility, the same way you do—but of course that's not the case.
+
+As you might imagine,
+some color combinations that are easy for some people to read are difficult or impossible for others.
+This usually comes down to color contrast,
+the relationship between the foreground and background colors' luminance.
+When the colors are similar, the contrast ratio is low;
+when they are different, the contrast ratio is high.
+
+The [WebAIM guidelines](https://webaim.org/standards/wcag/) recommend an AA (minimum) contrast ratio of 4.5:1 for all text.
+An exception is made for very large text (120-150% larger than the default body text),
+for which the ratio can go down to 3:1. Notice the difference in the contrast ratios shown below.
+
+<figure class="w-figure">
+  <img src="./contrast-ratios.jpg" alt="An image showing the different contrast ratios">
+</figure>
+
+The contrast ratio of 4.5:1 was chosen for level AA
+because it compensates for the loss in contrast sensitivity
+usually experienced by users with vision loss equivalent to approximately 20/40 vision.
+20/40 is commonly reported as typical visual acuity of people at about age 80.
+For users with low vision impairments or color deficiencies,
+we can increase the contrast up to 7:1 for body text.
+
+You can use the Accessibility Audit in Lighthouse to check your color contrast.
+Open DevTools, click Audits, and select Accessibility to run the report.
+
+<figure class="w-figure">
+  <img src="./accessibility-audit.jpg" alt="An image showing the different contrast ratios">
```

Comment:
Since Lighthouse just released version 6 we may want to update these screenshots to the latest. I think we could use any website for the screenshots (maybe a glitch that we put together so as not to shame anyone)

Reply:
Here's a screenshot you can use

![image](https://user-images.githubusercontent.com/4713486/83206973-fa556e80-a106-11ea-9cae-3373d3020463.png)

Reply:
@robdodson now that I compare Rachel's screenshot with the one I just took on Canary, I think she was already using LH v6

Reply:
oh ok. I thought they might have been from the original article.

Reply:
Any that were DevTools I redid, as the info was out of date with the recent changes.